### PR TITLE
Add core weekly 42

### PR DESCRIPTION
**Pull request by matks**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Add Core Weekly 42
| Fixed ticket? |

### CODE REVIEW

**Comment by matks** [comment]
To be merged tomorrow as it's already 6:30pm (so I put tomorrow as publication date)

File: news/_posts/core-weekly/2020-10-19-coreweekly-42-2020.md [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+In the meantime, all the `major` and `critical` bugs reported in PrestaShop 1.7.7.0 Release Candidate 1 first build have been fixed in the `1.7.7.x` branch. A new build has been delivered to QA team for, once again, testing before public release. Well, the testing will probably start tomorrow as QA team will be busy with the PrestaShop Day event.
+
+If you can't wait to test PrestaShop 1.7.7.0 Release Candidate, you can warm up by testing the [php8.0.0 Release Candidate 2](https://www.php.net/archive/2020.php#2020-10-16-1) published on the 16th of October 2020.
+
+
+## Releases
+
+* [dashtrends](https://github.com/PrestaShop/dashtrends) module: [v2.0.3](https://github.com/PrestaShop/dashtrends/releases/tag/v2.0.3)
+* [welcome](https://github.com/PrestaShop/welcome) module: [v6.0.2](https://github.com/PrestaShop/welcome/releases/tag/v6.0.2)
+* [ps_checkpayment](https://github.com/PrestaShop/ps_checkpayment) module: [v2.0.5](https://github.com/PrestaShop/ps_checkpayment/releases/tag/v2.0.5)
```
Comment (Progi1984) [inline news/_posts/core-weekly/2020-10-19-coreweekly-42-2020.md]:
```suggestion
* [ps_checkpayment](https://github.com/PrestaShop/ps_checkpayment) module: [v2.0.5](https://github.com/PrestaShop/ps_checkpayment/releases/tag/v2.0.5)
* [welcome](https://github.com/PrestaShop/welcome) module: [v6.0.2](https://github.com/PrestaShop/welcome/releases/tag/v6.0.2)
```

Comment (matks) [reply news/_posts/core-weekly/2020-10-19-coreweekly-42-2020.md]: I dont see the logic but if that makes you happy 😅

Comment (Progi1984) [reply news/_posts/core-weekly/2020-10-19-coreweekly-42-2020.md]: The logic is the alphabetic order :rofl:

**Comment by matks** [comment]
Thank you for review

**Review by Progi1984 — APPROVED** [review]

File: news/_posts/core-weekly/2020-10-19-coreweekly-42-2020.md (hunk 1 above)
Comment (matks) [reply news/_posts/core-weekly/2020-10-19-coreweekly-42-2020.md]: I retrieve this list from GitHub API 🤔  so I guess there must be an order (not the alphabetical). Maybe by release date ?

**Pull request merged**